tests/regression/tests/REQUEST-922-MULTIPART-ATTACK/922130.yaml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Meanwhile  I modified the test file, I suggest you to update this. See the new [one](https://github.com/coreruleset/coreruleset/pull/3796/files#diff-7356b747daef268f00c76bc370387da08ab20fbf8bc5a45462d4ada2f98e03bdR1)